lib/chef_zero/endpoints/node_identifiers_endpoint.rb

```diff
@@ -0,0 +1,22 @@
+require 'ffi_yajl'
+require 'chef_zero/rest_base'
+require 'uuidtools'
+
+module ChefZero
+  module Endpoints
+    # /organizations/NAME/nodes/NAME/_identifiers
+    class NodeIdentifiersEndpoint < RestBase
```

Comment:
I looked at the server code and the pedant test cases.  This endpoint is implemented in https://github.com/chef/chef-server/blob/master/src/oc_erchef/apps/oc_chef_wm/src/chef_wm_object_identifiers.erl.

FWIW, I'm not super happy with this implementation, it was meant as a stub so that tests pass.  I would prefer that someone with more experience with Chef Server code could review and/or update to ensure it is correct.